bpf/cpu/cpu.bpf.c

```diff
@@ -517,7 +518,7 @@ static __always_inline void add_stack(struct bpf_perf_event_data *ctx, u64 pid_t
     }
 
   } else if (method == STACK_WALKING_METHOD_FP) {
-    bpf_printk("[info] fp unwinding %d", DISABLE_BPF_HELPER_FP_UNWINDER);
+    // bpf_printk("[info] fp unwinding %d", DISABLE_BPF_HELPER_FP_UNWINDER);
```

Comment:
❓ Was this noisy?

Reply:
Very! Every single time we fetch a stack using the fp unwinding (e.g. Go processes) we would get a log